tools/flaky-test-retryer/handler.go

```diff
@@ -88,8 +90,36 @@ func (hc *HandlerClient) Listen() {
 // HandleMessage gets the job's failed tests and the current flaky tests,
 // compares them, and triggers a retest if all the failed tests are flaky.
 func (hc *HandlerClient) HandleMessage(msg *prowapi.ReportMessage) {
-	log.Println("Job fit all criteria - Starting analysis")
-	// TODO: ensure job failed due to tests, get job's failed tests, get current
-	//       flaky tests, cross-reference failed tests and flaky tests, retry tests
-	//       if all failed tests are flaky.
+
+	prefix := fmt.Sprintf("%s/pull/%d: ", msg.Refs[0].Repo, msg.Refs[0].Pulls[0].Number)
```

Comment:
It would be good to include job name here as well

Reply:
done